src/EditorFeatures/Core/Implementation/RenameTracking/RenameTrackingTaggerProvider.TrackingSession.cs

```diff
@@ -179,6 +179,13 @@ public void Cancel()
                         }
                         else
                         {
+                            // We do not yet support renaming (inline rename or rename tracking) on
```

Comment:
It reuses a lot of the same logic via `RenameUtilities.GetTokenRenameInfo`, but perhaps more could be unified, yes.